import { Control } from '../../Control.js';
export declare function captureError(fn: () => unknown): {
    didThrow: boolean;
    error: unknown;
    result: unknown;
};
export declare function captureAsyncError(fn: () => Promise<void>): Promise<{
    didThrow: boolean;
    error: unknown;
    result: unknown;
}>;
export declare function processError(control: Control, thrownError: unknown, sentenceStart: string, errorClassOrMessage?: (new (...args: any[]) => Error) | string | RegExp, message?: string | RegExp): void;
//# sourceMappingURL=errors.d.ts.map